controllers/team_controller.py

```diff
@@ -11,6 +11,21 @@
 from renderers.team_renderer import TeamRenderer
 from template_engine import jinja2_engine
 
+import webapp2
+import random
+
+
+class RandomTeam(webapp2.RequestHandler):
+    """
+    Lovingly written by Caleb Denio (https://github.com/cjdenio)
+    """
+    def get(self):
+        teams = Team.query().fetch(keys_only=True)
```

Comment:
We should probably put this behind memcache for speed and $ reasons.

Reply:
I'm open to changes, though I'm not sure how a random response can be cached.

Reply:
Pretty sure the "all teams" query is what you'd want to cache.

Reply:
I'm not sure what our caching capacity is, but caching *every single team* sounds like a bit much. Could this leech off of the cache of the [team list page](https://www.thebluealliance.com/teams)?